### ISSUE TITLE
Works strange after update or restarting pc

### ISSUE THREAD

**Issue by ViStatera** [Rem0o/FanControl.Releases#1082]
It worked fine and then after updating there is a problem and even if I don't update the problem still exists. If I set up any curve or do manual control the case fans max speed is 800 Rpm( max is 2000) and the Aio fans don't work at all. If I leave it at default it works fine
![image](https://user-images.githubusercontent.com/63738053/188578680-dd3dc758-8714-4a3c-8b69-c74eda261765.png)
![image](https://user-images.githubusercontent.com/63738053/188578850-77975ba3-f82a-4e27-ad8d-af884609ecb2.png)

**Comment by Rem0o** [comment]
Let me guess, Asus board?

**Comment by ViStatera** [comment]
> Let me guess, Asus board?

Yeah, you are right. I was going to delete the asus and corsair software anyway, but only after I set up all speed fans. I uninstalled it all in the morning( just couldn't wait) and yeah it started working properly. However, I do not understand why it was working the whole week without problems( with asus software) and after update it didn't anymore.

**Comment by Rem0o** [comment]
The asus software can change the "available" rpm range. 
Like if you set it "quiet" mode, 100% on the fan will be something like 50% of the actual max RPM available.

**Comment by ViStatera** [comment]
Strange that it worked all well before today

**Comment by Rem0o** [comment]
If you had asus software running before, call it unexpected behavior. 
The fact is was working before was more of a happy coincidence than a certitude.

**Comment by henrikhermansen** [comment]
This sounds a lot like my problem. Everything was working great until some recent update (can't remember which version I had, and I skipped a few versions when I went to 127).
I have ASUS board, and always had FanXpert installed, but disabled. I have now tried uninstalling FanXpert, but it didn't help.

After I boot up, the fans are not controlled reliably. Speed is shown at some weird value (sometimes the % is wrong, sometimes the rpm is wrong), and the speed doesn't change when the temperature on the curve changes. It mainly affects the fans, but has affected the pump a couple of times too. A question mark shows up sometimes indicating some other software is taking control, but Force apply doesn't seem to help either. I can't recall seeing the question mark until this recent update.

![image](https://user-images.githubusercontent.com/435037/189727796-5c4cfad5-caca-43c3-a66f-ecc8575e4e5a.png)
In the picture there are currently no question marks, but the speed percent on Bottom and top is showing 0%. The RPM sensor seems fine atm.
So far I can always fix it by running Refresh sensors detection one or a few times, but it's a hassle to remember to do it.

It seems strange that it worked fine for both me and ViStatera before, and now it's recently been broken for both of us. Perhaps you could look more into what has happened since V120? (I'm sure I was on 120-something before going to 127)

Could the AsusWMI plugin be a solution here? Or is that meant for some other purpose/problem?
Is there anything else we can try, or logs we can pull to investigate the issue?

**Comment by Rem0o** [comment]
@henrikhermansen 
Expand your control cards and repost the same screenshot.

**Comment by henrikhermansen** [comment]
@Rem0o I previously posted right after booting. Since then I have refreshed and fixed the fan controls. If this means the below screenshot is of no help, I can post another one the next time I have booted up my computer.
![image](https://user-images.githubusercontent.com/435037/189735829-6664c80a-9b87-40bc-b7f1-19d8a397e4cf.png)

**Comment by Rem0o** [comment]
@henrikhermansen  No that answer my suspicion.

Remove the start% from that control card ( set it to 0 ). That header can't be stopped.

**Comment by henrikhermansen** [comment]
@Rem0o this has me a little confused, so I hope you don't mind me asking some questions to get a better understanding.

I assume you mean the Bottom and top fans now, but what makes you say the header can't be stopped? I believe the Start% was set when I ran some automated setup with your software. These are two sets of Lian Li UNI Fans, but I can't remember exactly how I made the connections to L-Connect hub and/or motherboard.
However, I don't want them to stop, so I made the fan curve as it is.
Yet another however, even if I change the curve or use manual control, I can't make them stop. Which is strange, because I remember having stopped them at some point in the past.

I am also wondering about the Side fans and Water pump. As I mentioned, the pump as only been a problem once (that I have noticed), but the Side fans has also had issues where they just stayed off when the temperature rose up way beyond the point at which they should start spinning. These fans also worked flawlessly in the past. They are connected through a Y-split directly to the mobo.

**Comment by Rem0o** [comment]
Basically, if fan control can't stop a fan (  0 RPM @ 0%  ), start and stop % should be left at 0.  Otherwise, you may get weird behavior when a fan curve sets below the start/stop %.  

Maybe I should enforce that more. I've refactored some code that manages all that. I assumed that a fan configured with start/stop does stop in my refactor, which was wrong because some users have start/stop set on fan that don't stop, even if that doesn't make sense. That case is now covered in a future update.

> Which is strange, because I remember having stopped them at some point in the past.

To make sure, just set all of your controls to manual mode, slide them to 0, and see if they stop.  I don't see anything that would have made them stop before and not now from FanControl's perpective.

**Comment by henrikhermansen** [comment]
Thanks, that does make sense. Perhaps I have unknowingly changed something else at some point since that time I made top and bottom stop.

However, I'm still wondering about the Side fans. They were fine before, but I've had issues with those recently as well, and they actually can be stopped, so I think their settings are all right.

**Comment by Rem0o** [comment]
Update to V128.

**Comment by henrikhermansen** [comment]
@Rem0o V128 introduced some strange bug for me. Every time I boot up my computer, and then open FanControl, I see way too many fans. Also, you can see they don't follow their assigned curves.
![image](https://user-images.githubusercontent.com/435037/190624079-d5f50edf-380f-4636-af35-62a56b7b8599.png)

I can fix it by loading my configuration, but before I can do that, I get a message that my configuration has changed, and I'm asked to save or discard. I hit discard, and load my config, and all seems to be working fine now.
It seems something weird is happening every time the application launches after boot.

**Comment by Rem0o** [comment]
Fixed in V129.
